### PR TITLE
Fix/ux ui flow

### PR DESCRIPTION
**Pull request by AlexeyAndryukevich**

### CODE REVIEW

File: frontend/src/app/modules/project/components/create-project/create-project.component.ts [hunk 1]
```diff
@@ -130,12 +131,19 @@ export class CreateProjectComponent implements OnInit, OnDestroy {
...
-        : (isMaxError)
-          ? `Quantity must be less than ${control.errors.max.max}!`
-          : 'validation error';
+        const control = this.projectForm.get(field);
+        const isMaxError: boolean = !!control.errors && !!control.errors.max;
+        const isMinLengthError: boolean = !!control.errors && !!control.errors.minlength;
+        const isMaxLengthError: boolean = !!control.errors && !!control.errors.maxlength;
+
+        return control.hasError('required')
+                ? 'Value is required!'
```
Comment (Potapy4) [inline frontend/src/app/modules/project/components/create-project/create-project.component.ts]: Please, rewrite this to switch/case or if/else block

File: frontend/src/app/modules/project/components/project-settings/project-settings.component.ts [hunk 2]
```diff
@@ -73,54 +73,61 @@ export class ProjectSettingsComponent implements OnInit, OnDestroy {
...
+        this.isDetailsSaved = false;
+        this.projectService.updateProject(this.project)
+        .subscribe(
+            (resp) => {
+            this.router.navigate([`project/${this.projectId}`]);
+            this.isDetailsSaved = true;
+            this.toastService.success('New details have successfully saved!');
+            },
+            (error) => {
+            this.isDetailsSaved = true;
```
Comment (Potapy4) [inline frontend/src/app/modules/project/components/project-settings/project-settings.component.ts]: Should be false on error?

File: frontend/src/app/modules/project/components/project-settings/project-settings.component.ts [hunk 3]
```diff
@@ -73,54 +73,61 @@ export class ProjectSettingsComponent implements OnInit, OnDestroy {
...
-    const control = this.projectForm.get(field);
-    const isMaxError: boolean = !!control.errors && !!control.errors.max;
-    return control.hasError('required')
-      ? 'Value is required!'
-      : (isMaxError)
-        ? `Quantity must be less than ${control.errors.max.max}!`
-        : 'validation error';
-  }
+        return control.hasError('required')
+                ? 'Value is required!'
```
Comment (Potapy4) [inline frontend/src/app/modules/project/components/project-settings/project-settings.component.ts]: Same here, need to rewrite

File: frontend/src/app/modules/project/components/project-settings/project-settings.component.ts (hunk 2 above)
Comment (AlexeyAndryukevich) [reply frontend/src/app/modules/project/components/project-settings/project-settings.component.ts]: Nop, it's must be false, this variable is for spinner. I am agree it's not obvious name. I'll change on "HasDetailsSaveResponse"

File: frontend/src/app/modules/project/components/create-project/create-project.component.ts (hunk 1 above)
Comment (AlexeyAndryukevich) [reply frontend/src/app/modules/project/components/create-project/create-project.component.ts]: I thought about "switch", but did not find the corresponding enum. I'll try again.

Comment (AlexeyAndryukevich) [reply frontend/src/app/modules/project/components/create-project/create-project.component.ts]: I did it with "If/else". Check it!)

**Pull request merged**